src/components/sessions/SessionTimeline.tsx

```diff
@@ -67,6 +69,19 @@ function getMessage(intl, num: number, max: number): any {
   }
 }
 
+const calculateRuntime = ({ startedAt }): string => {
+  const duration = moment.duration(moment().diff(startedAt))
```

Comment:
Use dayjs here instead of installing the "big" moment library for this single line of code. Dayjs can do exactly the same and is already installed as a dep (and you have used it below as well) ;)

Reply:
Makes sense, changed to dayjs library now and removed the moment library from the dependencies.